elements/pl-order-blocks/dag_checker_test.py

```diff
@@ -35,15 +34,11 @@
 ]
 
 
-class TestDAGChecker(unittest.TestCase):
+class TestDAGChecker:
```

Comment:
Wrapping tests in a class is unnecessary with `pytest` - let's make `test_grade_dag` a top-level function.